scripts/diamond_clean_up.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
Yest, of course, but we'll keep it here until our diamond plugin uninstall operation gets merged